tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/interceptor/LateFtInterceptor.java

```diff
@@ -0,0 +1,69 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.eclipse.microprofile.fault.tolerance.tck.interceptor;
+
+
+import org.eclipse.microprofile.fault.tolerance.tck.interceptor.CounterFactory.CounterId;
+import org.eclipse.microprofile.fault.tolerance.tck.interceptor.LateFtInterceptor.InterceptLate;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import javax.annotation.Priority;
+import javax.inject.Inject;
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.Interceptor;
+import javax.interceptor.InterceptorBinding;
+import javax.interceptor.InvocationContext;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * An interceptor that is called after the FT interceptor
+ * in the chain and count the invocation.
+ * @author carlosdlr
+ */
+@Interceptor
+@InterceptLate
+@Priority(Interceptor.Priority.LIBRARY_AFTER)
```

Comment:
In #313 we decided that the FT interceptor would have priority `PLATFORM_AFTER+10`. The LateFtInterceptor needs to have a priority larger than this for the test to work.

`LIBRARY_AFTER = 3000`, `PLATFORM_AFTER = 4000`

Reply:
@Azquelt But this FT ordering is already implemented? because if you executed this test, for example, in liberty container the asserts are working good. Can you help me to clarify this?
Thanks

Reply:
@carlosdlr We have to update the initial interceptor order when we implement FT 2.0. The tests you are adding is for FT 2.0.

Reply:
I originally wrote the test against the OpenLiberty 1.1 implementation, which set the priority as `LIBRARY_BEFORE`. So the interceptor priorities were correct for OpenLiberty's FT 1.1 support, but, as Emily said, they're not correct for what we specified for 2.0.

Reply:
This was fixed changing the later interceptor priority to PLATFORM_AFTER+100 after the FT priority which is  PLATFORM_AFTER+10